wazimap/models/data.py

```diff
@@ -967,7 +967,7 @@ def for_fields(cls, fields, universe=None, dataset=None, name=None):
 
         candidates = cls.objects.filter(fields__contains=list(field_set))
         if name:
-            candidates = candidates.filter(name=name)
+            candidates = candidates.filter(name=name.upper())
```

Comment:
how about `name__iexact`, just to be safe?

Reply:
Cool, that's much better. I've updated it.